doc/api/fs.md

```diff
@@ -685,6 +685,33 @@ added: v10.0.0
 
 Changes the ownership of a file.
 
+### `fsPromises.copy(src, dest[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `src` {string|Buffer|URL} source path to copy.
+* `dest` {string|Buffer|URL} destination path to copy to.
```

Comment:
Do we need tests for when `src` and/or `dest` is a `Buffer` or a `URL`?

Reply:
Added test:

https://github.com/nodejs/node/pull/39372/commits/30297360a6d41922568f556abe50f13248311ef7